packages/react-components/src/components/Icon/Icon.tsx

```diff
@@ -46,6 +53,14 @@ const IconSizeMap: Record<IconSize, { width: number; height: number }> = {
     width: 32,
     height: 32,
   },
+  xxlarge: {
+    width: 48,
+    height: 48,
+  },
+  xxxlarge: {
+    width: 64,
+    height: 64,
+  },
```

Comment:
I think we can add some sort of table in our Icon page, to illustrate IconSize name and icon dimensions.